packages/api/test.tap

```diff
@@ -0,0 +1,3 @@
+1..70
```

Comment:
whoops, we should delete this file, not sure how it got checked in (_I added an ignore rule to .gitignore)_.